components/templates/template.html.twig

```diff
@@ -1,818 +1,2350 @@
+                                           @keyup.enter="saveTranslationUpdate('metatags', cell)"/>
+                                    <span class="absolute fa fa-tags"></span>
+                                    <input class="table_input" v-model:value="cell.row['alias_pt']" type="text"
+                                           @input="stashUpdate('metatags', cell)"
+                                           @keyup.enter="saveTranslationUpdate('metatags', cell)"/>
+                                </div>
+                            </template>
+                            <template slot="ru" scope="cell">
+                                <div v-if="cell.row['is_active']=='1'">
+                                    <span class="absolute flag-icon flag-icon-ru"></span>
+                                    <input class="table_input" v-model:value="cell.row['ru']" type="text"
+                                           @input="stashUpdate('metatags', cell)"
+                                           @keyup.enter="saveTranslationUpdate('metatags', cell)"/>
+                                    <span class="absolute fa fa-tags"></span>
+                                    <input class="table_input" v-model:value="cell.row['alias_ru']" type="text"
+                                           @input="stashUpdate('metatags', cell)"
+                                           @keyup.enter="saveTranslationUpdate('metatags', cell)"/>
+                                </div>
+                            </template>
+                            <template slot="sv" scope="cell">
+                                <div v-if="cell.row['is_active']=='1'">
+                                    <span class="absolute flag-icon flag-icon-se"></span>
+                                    <input class="table_input" v-model:value="cell.row['sv']" type="text"
+                                           @input="stashUpdate('metatags', cell)"
+                                           @keyup.enter="saveTranslationUpdate('metatags', cell)"/>
+                                    <span class="absolute fa fa-tags"></span>
+                                    <input class="table_input" v-model:value="cell.row['alias_sv']" type="text"
+                                           @input="stashUpdate('metatags', cell)"
+                                           @keyup.enter="saveTranslationUpdate('metatags', cell)"/>
+                                </div>
+                            </template>
+                            <template slot="cs" scope="cell">
+                                <div v-if="cell.row['is_active']=='1'">
+                                    <span class="absolute flag-icon flag-icon-cz"></span>
+                                    <input class="table_input" v-model:value="cell.row['cs']" type="text"
+                                           @input="stashUpdate('metatags', cell)"
+                                           @keyup
```

Comment:
Why is basically all the template changed? you should have minimal changes in here, can you please check if you changed the spaces to tabs or the encoding. A proper review is like that hardly possible